Sources/Widgets/Widgets3D/ResliceCursorWidget/behavior.js

```diff
@@ -68,6 +77,7 @@ export default function widgetBehavior(publicAPI, model) {
     }
     if (isScrolling) {
       if (model.previousPosition.y !== callData.position.y) {
+        publicAPI.updateState();
```

Comment:
why not call it on `handleLeftButtonPress` instead?

Reply:
I just updated where I call this method. We don't want to call it each time we click (it's already done by the representation) but only when we scroll.
So, I called this method when the attribute `isScrolling` is set to true.